### DISCUSSION TITLE
Advanced documentation about declarativenetrequest api, its regexfilter, transform, regexSubstituion

### DISCUSSION THREAD

**Discussion by libreom** [uBlockOrigin/uBOL-home#95] [Q&A]
If anyone has advanced and detailed documentation or guide about these, please point https://developer.chrome.com/docs/extensions/reference/api/declarativeNetRequest seems basic and mdn too
Thanks

**Comment by ghost** [comment]
https://github.com/w3c/webextensions/issues/344
